nvme-print.c

```diff
@@ -208,8 +208,8 @@ static void json_nvme_id_ns(struct nvme_id_ns *ns, unsigned int mode)
 	json_free_object(root);
 }
 
-static void json_nvme_id_ctrl(struct nvme_id_ctrl *ctrl, unsigned int mode,
-			void (*vs)(__u8 *vs, struct json_object *root))
+static void json_nvme_id_ctrl(struct nvme_id_ctrl *ctrl, void (*vs)(__u8 *vs,
```

Comment:
This is not a very good way to break up this line. Otherwise, looks fine.

Reply:
modified that line and pushed again!